client.go

```diff
@@ -1738,26 +1741,28 @@ func (c *ToznySDKV3) GetOrCreateNamespace(ctx context.Context, options Namespace
 			return nil, err
 		}
 		group = createGroupResponse
-		// get membership keys
-		membershipKeyRequest := storageClient.CreateMembershipKeyRequest{
-			GroupAdminID:      c.StorageClient.ClientID,
-			NewMemberID:       c.StorageClient.ClientID,
-			EncryptedGroupKey: createGroupResponse.EncryptedGroupKey,
-			ShareePublicKey:   c.StorageClient.EncryptionKeys.Public.Material,
-		}
-		membershipKeyResp, err := c.StorageClient.CreateGroupMembershipKey(ctx, membershipKeyRequest)
-		if err != nil {
-			return nil, err
-		}
-		// make the capabilities
+		// make the capabilities -- for now, all members get read and share capabilities
 		groupMemberCapabilities := []string{storageClient.ShareContentGroupCapability, storageClient.ReadContentGroupCapability}
```

Comment:
Is this more permissive than we want for the default behavior. It seems like group members by default should be read only and then can be promoted to sharer.